entity-framework/core/miscellaneous/context-pooling.md

```diff
@@ -0,0 +1,40 @@
+---
+title: DbContext Pooling
+description: DbContext pooling in Entity Framework Core
+author: rick-anderson
+ms.author: riande
+ms.date: 9/19/2020
+uid: core/miscellaneous/dbcontextpool
+---
+# DbContext pooling
+
+`AddDbContextPool` enables pooling of DbContext instances. Context pooling can increase throughput in high-scale scenarios such as web servers by reusing `DbContext` instances, rather than creating new instances for each request.
```

Comment:
```suggestion
`AddDbContextPool` enables pooling of `DbContext` instances. Context pooling can increase throughput in high-scale scenarios such as web servers by reusing context instances, rather than creating new instances for each request.
```